tests/robot-tests/tests/admin/bau/prerelease_and_amend.robot

```diff
@@ -0,0 +1,302 @@
+
+    user checks list has x items    testid:invitableList    2    ${modal}
+    user checks list item contains    testid:invitableList    1    simulate-delivered-1@notifications.service.gov.uk
+    ...    ${modal}
+    user checks list item contains    testid:invitableList    2    simulate-delivered-2@notifications.service.gov.uk
+    ...    ${modal}
+    user clicks button    Confirm
+
+Go to "Sign off" page again
+    user clicks link    Sign off
+    user waits until h2 is visible    Sign off
+    user waits until page contains button    Edit release status
+
+Approve release again
+    user clicks link    Sign off
+    user waits until h2 is visible    Sign off
+    user waits until page contains button    Edit release status
+    user clicks button    Edit release status
+    user waits until h2 is visible    Edit release status
+    user clicks radio    Approved for publication
+    user enters text into element    id:releaseStatusForm-internalReleaseNote    Approved by UI tests
+    user clicks radio    On a specific date
+    user waits until page contains
+    ...    Pre-release users will have access to a preview of the release 24 hours before the scheduled publish date.
+    user clicks radio    Immediately
+    user checks page contains
+    ...    Pre-release users will not have access to a preview of the release if it is published immediately.
+    user clicks button    Update status
+    user waits until h2 is visible    Sign off
+    user checks summary list contains    Current status    Approved
+
+user creates amendment for release
+    [Arguments]    ${PUBLICATION_NAME}    ${RELEASE_NAME}
+    user navigates to publication page from dashboard    ${PUBLICATION_NAME}
+
+    ${ROW}=    user gets table row    ${RELEASE_NAME}    testid:publication-published-releases
+    user clicks element    xpath://*[text()="Amend"]    ${ROW}
+
+    ${modal}=    user waits until modal is visible    Confirm you want to amend this published release
+    user clicks button    Confirm    ${modal}
+    user waits until page contains title    ${PUBLICATION_NAME}
+    user waits until page contains title caption    Amend release for ${RELEASE_NAME}
+    user checks page contains tag    Amendment
+
+Add basic release content
+    user clicks link    Content
+    user waits until h1 is visible    ${PUBLICATION_NAME}
+    user waits until h2 is visible    ${PUBLICATION_NAME}
+    user adds basic release content    ${PUBLICATION_NAME}
+
+Add release note to amendment
+    user clicks button    Add note
+    user enters text into element    id:createReleaseNoteForm-reason    Test release note
+    user clicks button    Save note
+    ${date}=    get current datetime    ${DATE_FORMAT_MEDIUM}
+    user waits until element contains    css:#releaseNotes li:nth-of-type(1) time    ${date}
+    user waits until element contains    css:#releaseNotes li:nth-of-type(1) p    Test release note
+
+Check Pre-release Access page during amendment
```

Comment:
Maybe to be more explicit about what we're testing here, we could rename this from `Check Pre-release Access page during amendment` to something like `Check that Pre-release users and Public access list are empty in new amendment`